frontend-web/src/lib/apiClient.ts

```diff
@@ -0,0 +1,24 @@
+export async function fetchFromApi(
+  url: string,
+  method: 'GET' | 'POST' | 'PUT' | 'DELETE' = 'GET',
+  body?: any
+): Promise<any> {
+  const options: RequestInit = {
+    method,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    cache: 'no-store',
+  };
+
+  if (body && method !== 'GET') {
+    options.body = JSON.stringify(body);
+  }
+
+  const response = await fetch(url, options);
```

Comment:
```suggestion
  const baseUrl = process.env.HTTP +. process.env.API_URL + ":" + process.env.PORT
  const response = await fetch(baseUrl + slug, options);
```

L'idée est que ton client se charge de récupérer toutes les informations possibles à partir d'un slug.
Il sait deja comment contacter l'api, s'authentifier, etc.
Pas besoin que le composant qui l'appelle se charge de ca.